docs/fastdds/xml_configuration/common.rst

```diff
@@ -0,0 +1,416 @@
+.. include:: includes/aliases.rst
+
+.. _commonxml:
+
+Common
+------
+
+The preceding XML profiles define some XML elements that are common to several profiles.
+This section aims to explain these common elements.
+
+*   :ref:`LocatorListType`
+*   :ref:`PropertiesPolicyType`
+*   :ref:`DurationType`
+*   :ref:`TopicType`
+
+    -   :ref:`hQos`
+    -   :ref:`rLsQos`
+
+*   :ref:`CommonQOS`
+
+    -   :ref:`xml_durability`
+    -   :ref:`xml_liveliness`
+    -   :ref:`xml_partition`
+    -   :ref:`xml_deadline`
+    -   :ref:`xml_lifespan`
+    -   :ref:`xml_disablepositiveacks`
+    -   :ref:`xml_latencybudget`
+
+*   :ref:`Throughput`
+*   :ref:`CommonAlloc`
+
+
+.. _LocatorListType:
+
+LocatorListType
+^^^^^^^^^^^^^^^
+
+It represents a list of |Locator_t|.
+LocatorListType is used inside other configuration parameter labels that expect a list of locators,
+for example, in ``<defaultUnicastLocatorList>``.
+Therefore, LocatorListType is defined as a set of ``<locator>`` elements.
+The ``<locator>`` element has a single child element that define the transport protocol for which the locator is
+defined. These are: ``<udpv4>``, ``<tcpv4>``, ``<udpv6>``, and ``<tcpv6>``.
```

Comment:
Maybe a note clarifying that SHM locators cannot be configured, as SHM handles them automatically?